src/conf.c

```diff
@@ -162,7 +162,7 @@ void conf_setDefaults (void)
    conf.font_size_console = 10;
    conf.font_size_intro   = 18;
    conf.font_size_def     = 12;
-   conf.font_size_small   = 11;
+   conf.font_size_small   = 11.5;
```

Comment:
This is a bit scary because there's probably a whole amount of minor visualization glitches that may appear when changing the font size. Is the difference between 12 and 11.5 significant enough to warrant two different fonts?